pkg/paralleltest/testdata/src/ignoremissingsubtests/ignoremissingsubtests_test.go

```diff
@@ -0,0 +1,163 @@
+package t
```

Comment:
This is just a duplicate of `t_test.go` with some of the expected warnings removed.